packages/wp-story-editor/src/api/media.js

```diff
@@ -69,6 +69,19 @@ export function getMedia(
   }));
 }
 
+// Important: Keep in sync with REST API preloading definition.
+export function getMediaForCorsCheck(config) {
+  const path = addQueryArgs(config.api.media, {
+    context: 'edit',
+    per_page: 10,
+    _fields: 'source_url',
+  });
+
+  return apiFetch({ path }).then((attachments) =>
+    attachments.map((attachment) => attachment?.source_url)
```

Comment:
Nit: `attachment` is not `null` here, so no need for optional chaining.

```suggestion
    attachments.map((attachment) => attachment.source_url)
```